wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@mikekaganski not exactly, but I could define the buffer to be a string full of "\0":

```julia
message_buffer = repeat("\0",buffer_length)
function get_parameter_information_string(Key::AbstractString,OutType::AbstractString)
  message_buffer = OutType*message_buffer[length(OutType):end]
  val = ccall( (:get_parameter_information_string, "CoolProp"), Clong, (Ptr{UInt8},Ptr{UInt8},Int), Key,message_buffer::Array{UInt8,1},buffer_length)
  return bytestring(convert(Ptr{UInt8}, pointer(message_buffer::Array{UInt8,1})))
end
```

With the restriction that strings can't be modified, so I have to replace it with the two peaces of string (the `*` operator allows to attach two strings together. Julia is quite type sensitive)